modules/geo-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -17,7 +18,8 @@ function validateGeometryAttributes(attributes) {
 
 const defaultProps = {
   // PBR material object. _lighting must be pbr for this to work
-  pbrMaterial: {type: 'object', value: null}
+  pbrMaterial: {type: 'object', value: null},
+  featureIds: {type: 'array', value: null}
```

Comment:
This will fail the prop type check because `null` is not an array. Add `optional: true`.

Reply:
Fixed